docs/concepts/package-types.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Types of packages
+parent: Concepts
+---
+
+| Type       | `modular start` | `modular build` | `modular test` | `modular lint` | Custom index / assets      | Bundled                                      | Entrypoint                                                                                                                                                                                                                                                                                       |
+| ---------- | --------------- | --------------- | -------------- | -------------- | -------------------------- | -------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `app`      | Yes             | Yes             | Yes            | Yes            | Yes                        | Yes                                          | The compiled bundle will use `src/index.tsx` as an entrypoint and include it in the output `index.html`                                                                                                                                                                                          |
```

Comment:
Include in the _outputted_ index.html or include it in the _output of_ index.html?